react-cra/src/index.css

```diff
@@ -8,6 +8,11 @@ body,
     overflow-y: auto;
   }
 
+.App {
+    display: flex;
+    flex-direction: column;
+}
+
```

Comment:
Do we need these rules for both App and `<main>`?

Reply:
Yep, the same logic we have in the main, I have applied to the App wrapper, additional rule "flex: 1" was added to the main to make sure it takes all available space.
The main purpose is to push the footer down, otherwise, on a really long screen, the footer on the native view is dependent on the length of the content.